.gitignore

```diff
@@ -9,3 +9,6 @@ _ci-configs/*
 
 # Bazel directories
 cluster-provision/gocli/bazel-*
+
+# vim swap files
+*.sw*
```

Comment:
This should be in your own globalignore, not in the one for this project.

Reply:
Upsy